modules/compiler/cookie/{{cookiecutter.target_name}}/source/info.cpp

```diff
@@ -38,7 +38,7 @@ namespace {{cookiecutter.target_name}} {
   dma_optimizable = true;
   supports_deferred_compilation = false;
   scalable_vector_support = {{cookiecutter.scalable_vector}};
-  kernel_debug = false;
+  kernel_debug = true;
```

Comment:
Is this meant to stay in?

Reply:
yes, it fixes the part of check-ComputeAorta that tests with '-g'

Reply:
Ah, thanks, this is a confusing member name but its comment (kernel supports debugging, which I hadn't seen) helps explain.